test_pull_requests

```diff
@@ -124,12 +124,44 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_WAITING_IN_QUEUE = "Waiting: You are in the build queue at position:"
+CONTENT_WAITING_DETERMINING_QUEUE_POS = "Waiting: Determining build queue position"
+CONTENT_WAITING_STABLE_BUILD = "Waiting for stable build of"
+CONTENT_RUNNING = "Running"
+CONTENT_WAITING = "Waiting"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
+  CONTENT_EVALUATING => :evaluating,
+  CONTENT_RUNNING    => :running,
+  CONTENT_WAITING    => :waiting,
```

Comment:
Why are we keeping this implied aliasing around? Since it's out of scope of your PR to break the `"waiting" ` block of the `case/when` into multiple blocks, I don't think you should break the aliasing, but it makes sense at least to make it explicit:
```rb
CONTENT_TO_STATE = {
  ...
  CONTENT_WAITING_IN_QUEUE              => :waiting
  CONTENT_WAITING_DETERMINING_QUEUE_POS => :waiting
  CONTENT_WAITING_STABLE_BUILD          => :waiting
  ...
}
```